src/reports/package/footer-text-for-service.tsx

```diff
@@ -2,18 +2,27 @@
 // Licensed under the MIT License.
 import * as React from 'react';
 
-import { ReportSectionFactory } from 'reports/components/report-sections/report-section-factory';
+import { EnvironmentInfoProvider } from 'common/environment-info-provider';
+import { NamedFC } from 'common/react/named-fc';
 import { ToolLink } from 'reports/components/report-sections/tool-link';
 
+export type FooterTextForServiceProps = { environmentInfoProvider: EnvironmentInfoProvider };
```

Comment:
Original comment here:
`This is duplicated from 'src/reports/components/report-sections/footer-text.tsx', let's avoid this by defining the type in one place only and importing where we need to use it.`
The general idea is not about the name, it's about the how this components are related. Both are instances being use on `ReportSectionFactory.FooterText` with type `ReactFCWithDisplayName<Pick<SectionProps, 'environmentInfo'>>`; that's why the original code used `ReportSectionFactory['FooterText']`. That way, both comment refer to the same type, if you change the type, compilation will tell you you broke the two components. By creating a new type here (with a different or the same name but having the same properties for now) you blind yourself to this scenario, i.e. if something change on the type we are using on the other component, the compiler won't tell you you broke this component. You will only find out about the error when running the code. I don't think we have enough test coverage to catch that, since that's not something you can catch easily using unit tests and I'm pretty sure we don't have end to end test for exporting reports from AI-Web and the report package (I may be wrong).
My suggestion will be import `FooterTextProps` from `src\reports\components\report-sections\footer-text.tsx` that way we only have one type for the props object.